ThingIFSDK/ThingIFSDK/Gateway/GatewayAPIBuilder.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GatewayAPIBuilder.swift
+//  ThingIFSDK
+//
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+public class GatewayAPIBuilder {
+
+    private var tag: String?
+    private let app: App
+    private let gatewayAddress: NSURL
+    public var accessToken: String?
+
+    /** Set tag to this GatewayAPI instance.
+     tag is used to distinguish storage area of instance.
+     <br>
+     If the api instance is tagged with same string, It will be overwritten.
+     <br>
+     If the api instance is tagged with different string, Different key is used to store the instance.
+     <br>
+     <br>
+     Please refer to {@link GatewayAPI#loadFromStoredInstance(Context, String)} as well.
+
+     - Parameter tag: If null or empty String us passed, it will be ignored.
+     - Returns: builder instance for chaining call.
+     */
+    public func setTag(tag: String?) -> GatewayAPIBuilder
+    {
+        self.tag = tag
+        return self
+    }
+
```

Comment:
Let us remove this method, together with `accessToken` attribute from `GatewayAPIBuilder`